### ISSUE TITLE
Link to page with all participants

### ISSUE THREAD

**Issue by cbail** [compsocialscience/summer-institute#1250]
On the old site, we had a link that had a list of all the participants (i.e. students; not faculty, organizers, and teaching assistants) who have ever participated in any of the sites: 

https://compsocialscience.github.io/summer-institute/participants

Is it possible to create this page on the new site? In the long term we want to create a searchable database for the "People" page but in the short term this "participants" page is helpful to us because it let's us show funders how many students we reach. Thanks!

**Comment by msalganik** [comment]
One this new page "Participants" is created, the top nav should be changed from "People" to "Participants".

**Comment by msalganik** [comment]
Also, let's keep the people page, we just don't need any links to it.

**Comment by lukebaker** [comment]
Can you help me understand this request a bit more? My understanding is that the "People" page has replaced the "Participants" page. It sounds like you want a page that _only_ includes participants. However, to date we're not enforcing locations to define a list of participants. They can list their people in whichever groups they want to create, so we might not necessarily know which group(s) we should consider their participants. We could only search for a group called participants for this page if you'd like, but it might miss some people depending on how the location setup their people.

**Comment by msalganik** [comment]
That's correct.  We'd like a page that only includes participants.  That way we can point people to all the amazing participants we have had.

But, just looking through some of the files I can now see that folks did it differently different places.  I didn't realize that.  At Princeton we had our participants set apart (https://github.com/compsocialscience/summer-institute/blob/master/_data/2019/princeton/participants.yml) but at Oxford they did not (https://github.com/compsocialscience/summer-institute/blob/master/_data/2019/oxford/people.yml).  

Since this is complicated, let's just hold off.  We don't need this right now.

**Comment by lukebaker** [comment]
Yeah, there are two different ways for the partner locations to list their people. `people.yml` is the newest method and is more flexible than the old approach because they can name their groups whatever they want and sort those groups in the order that they choose. We could do a best-effort searching for the `participants.yml` or a `participants` section in `people.yml`. However, we're not guaranteed to that all locations will name their participants section `participants`.

**Comment by msalganik** [comment]
OK, let's put this on hold for now. We can come back to it later.
